api.md

```diff
@@ -58,6 +58,52 @@ your use case. Users will *rarely* want or need to use this command.
 converted to a hypertable (via `create_hypertable`), but must similarly
 be run only on an empty hypertable.
 
+**Space partitions**: The use of additional partitioning is a very
+specialized use case.  **Most users will not need to use it.**
```

Comment:
This might be out-of-context of this PR, but FWIW, I've never understood why we've always downplayed space partitioning. It must be beneficial in cases when you often query with equality, e.g.,`device = "foo"`, given that you space partition on `device`.

Not sure why we don't talk about that and only focus on multiple disks and parallelism? Also, for clustering, I think space partitioning will be more or less the recommended setup.

Reply:
I think the problem actually is that prior to our future releases, some query optimizations don't work with space partitions so people get weird performance results. I think we should go back in and update these once clustering is out, so this is just a temporary solution to make sure people don't assume that they need it.